### PR TITLE
HP-1520 Fix/ct gov

### PR DESCRIPTION
**Pull request by mfshao**
Link to JIRA ticket if there is one: https://ctds-planx.atlassian.net/browse/HP-1520

To support clinicaltrials.gov API V2 for HEAL CEDAR ingestion job

### Improvements
- HEAL CEDAR ingestion: don't update clinicaltrials.gov from CEDAR values

### CODE REVIEW

File: files/scripts/healdata/heal-cedar-data-ingest.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def update_filter_metadata(metadata_to_update):
 
 def get_client_token(client_id: str, client_secret: str):
     try:
-        token_url = f"http://revproxy-service/user/oauth2/token"
+        token_url = "http://revproxy-service/user/oauth2/token"
```
Comment (MichaelLukowski) [inline files/scripts/healdata/heal-cedar-data-ingest.py:117]: Can we change this to `https`? Snyk is throwing this url as an error. I know that this is hitting reverse proxy but I think that https will still probably work.

Comment (mfshao) [reply files/scripts/healdata/heal-cedar-data-ingest.py:117]: Yeah I wasn't sure if https would work, I was going to tell security this is a false-positive. But I can definitely test the https change

Comment (mfshao) [reply files/scripts/healdata/heal-cedar-data-ingest.py:117]: well, actually, it doesn't look like https gonna work for revproxy 😕 https://github.com/uc-cdis/cloud-automation/blob/master/kube/services/revproxy/revproxy-service.yaml#L8-L12

Comment (MichaelLukowski) [reply files/scripts/healdata/heal-cedar-data-ingest.py]: Yeah, had a small chat with Jawad about this and it won't work as handling certs for all the services would be a huge issue we would have to maintain.

**Review by MichaelLukowski — APPROVED** [review]
Everything looks good! Thanks for moving everything to the correct service rather than the revproxy.

**Pull request merged**